### PR TITLE
Dispatch event after having update the product

### PR DESCRIPTION
**Pull request by fitn**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md)

**Description (for Contributor and Core Developer)**

[//]: <> (What does this Pull Request do? reference the related issue?)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | OK
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Resources/config/updaters.yml [hunk 1]
```diff
@@ -72,6 +72,7 @@ services:
         arguments:
             - '@pim_catalog.updater.product_property_setter'
             - '@pim_catalog.updater.product_template'
+            - '@event_dispatcher'
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Resources/config/updaters.yml]: to add to the changelog please

**Comment by nidup** [comment]
Why add it only for product updater? We did the same in v1.5 with inconsistencies in events dispatched in different savers (pre and/or post sometimes, unitary and/or single sometimes) and we had to re-work all of them in v1.6 to allow to have a consistent behavior in extensions. I would prefer to add it for all and to document it properly. We already have cases in EE where we override because this event is missing so :+1: for the PR but imho only if applied averywhere. I'm available to discuss about it and to help :smile:

File: src/Pim/Component/Catalog/Updater/ProductUpdater.php [hunk 2]
```diff
@@ -139,6 +148,8 @@ public function update($product, array $data, array $options = [])
         }
         $this->updateProductVariantValues($product, $data);
 
+        $this->eventDispatcher->dispatch(ProductEvents::POST_UPDATE, new GenericEvent($product));
```
Comment (nidup) [inline src/Pim/Component/Catalog/Updater/ProductUpdater.php]: i would dispatch product + data in a specific event

**Comment by nidup** [comment]
known as TIP-675

**Comment by aRn0D** [comment]
@fitn What is the status of this PR? Will you add events everywhere?

**Comment by solivier** [comment]
@fitn any news for this PR ?

**Pull request closed without merging**